components/waitwhile/actions/list-customers/list-customers.ts

```diff
@@ -0,0 +1,86 @@
+import waitwhile from "../../app/waitwhile.app";
+import { defineAction } from "@pipedream/types";
+
+
+export default defineAction({
+  name: "List Customers",
+  version: "0.0.3",
```

Comment:
Should be version 0.0.1.
```suggestion
  version: "0.0.1",
```